code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -417,7 +418,10 @@
 	icon_state = "bamboo_sofamiddle"
 	color = null
 	colorable = FALSE
-
+	resistance_flags = FLAMMABLE
```

Comment:
any reason you added FLAMMABLE to these?

Reply:
yeah, removing that now. thanks for pointing that out.

Reply:
> any reason you added FLAMMABLE to these?

Well, forgot to ask about this, but they are technically made out of wood/bamboo, and bamboo is an easily flammable resource irl.

Reply:
agreed, its best to put these changes in the PR description

Reply:
updated the pr, thanks for the heads up